jdat_notebooks/background_estimation_imaging/Imaging Sky Background Estimation.ipynb

```diff
@@ -730,7 +775,9 @@
    "source": [
     "## Check for bias in due to the galaxies\n",
     "\n",
-    "Plot the value of the residual background under the central 3x3 pixels under each galaxy vs the flux of the galaxy, separately for those that are masked and those that are unmasked. First we need to get these values. Take the mean for  3x3 pixels centered on each source. Keep track of the ones that were masked in fitting the background and the ones that weren't so we can check separately for any bias."
+    "Since we are dealing with a simulation, where we know truth, we can evaluate the biase directly. (In the case of a real image, we can't do that; we'll suggest another test further down). \n",
```

Comment:
```suggestion
    "Since we are dealing with a simulation, where we know truth, we can evaluate the bias directly. (In the case of a real image, we can't do that; we'll suggest another test further down). \n",
```